pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -383,6 +387,25 @@ func (b *BridgeLibvirtSpecGenerator) generate() error {
 }
 
 func (b *BridgeLibvirtSpecGenerator) domainIfaceSpec() (*api.Interface, error) {
+	podNicLink, err := b.handler.LinkByName(b.podInterfaceName)
+	if err != nil {
+		log.Log.Reason(err).Errorf("failed to get a link for interface: %s", b.podInterfaceName)
+		return nil, err
+	}
+	if podNicLink.Type() == "dummy" {
```

Comment:
isn't there a const on netlink for this ? If not, please add one.

Reply:
No, [there](https://github.com/kubevirt/kubevirt/blob/825113495ad1b9e14886cbd884fef9fa8bdc7f47/vendor/github.com/vishvananda/netlink/link.go#L231) is no const.

The netlink repo is not under my control. Where exactly you meant I should add the const?

Anyway, changed to - `_, dummy := podNicLink.(*netlink.Dummy)`

Reply:
I meant you could have defined a const (somewhere) in kubevirt  - `const dummyIfaceType = "dummy"`

Then we'd argue about where it should go :P 

Your solution is better, thanks.